### ISSUE TITLE
Incorrect Terraform version being used when code is scanned by tfsec

### ISSUE THREAD

**Issue by mrmattyboy1** [MatthewJohn/terrareg#44]
Analytics are shown for tfsec when it scans modules

**Comment by MatthewJohn** [comment]
tfsec (v1.28.4) passes the following headers:
```
User-Agent: Go-http-client/1.1
X-Terraform-Version: 19.13.1
```

Terraform (1.4.5) passes the following headers:
```
User-Agent: Terraform/1.4.5
X-Terraform-Version: 1.4.5
```

I will look at ignoring X-Terraform-Version when the user agent does not conform to the `Terraform/(\d+)\.(\d+)\.(\d+)` format

**Comment by MatthewJohn** [comment]
This has been addressed in branch 519-ignore-terraform-version-from-tfsec-in-analytics

**Comment by MatthewJohn** [comment]
Created gitlab issue: https://gitlab.dockstudios.co.uk/pub/terrareg/-/issues/519
gitlab-issue-id:519

**Comment by MatthewJohn** [comment]
PR: https://gitlab.dockstudios.co.uk/pub/terrareg/-/merge_requests/404
This was fixed in 3.4.2 (b93a0846)
